frontend/public/module/k8s/get-resources.ts

```diff
@@ -4,13 +4,57 @@ import * as _ from 'lodash-es';
 
 import { coFetchJSON } from '../../co-fetch';
 import { K8sKind } from '../../module/k8s';
+import { API_DISCOVERY_RESOURCES_LOCAL_STORAGE_KEY } from '../../const';
 
 const ADMIN_RESOURCES = new Set(
   ['roles', 'rolebindings', 'clusterroles', 'clusterrolebindings', 'thirdpartyresources', 'nodes', 'secrets']
 );
 
 export const kindToAbbr = kind => (kind.replace(/[^A-Z]/g, '') || kind.toUpperCase()).slice(0, 3);
 
+export const cacheResources = resources => {
```

Comment:
We might want to make this return a `Promise` so we can use an async storage method (like [ServiceWorker cache](https://developer.mozilla.org/en-US/docs/Web/API/Cache)).